### PR TITLE
Custom domains

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: dash/orgs/views.py [hunk 1]
```diff
@@ -123,6 +123,8 @@ class OrgForm(forms.ModelForm):
 
     subdomain = forms.CharField(help_text=_("The subdomain of this organization"), required=False)
 
+    domain = forms.CharField(help_text=_("The subdomain of this organization"), required=False)
+
```
Comment (nicpottier) [inline dash/orgs/views.py]: Should there be some validation that this isn't set to the domain in settings.py?

Comment (norkans7) [inline dash/orgs/views.py]: I add that

**Comment by norkans7** [comment]
I added the validation and adjusted the link for org

**Comment by nicpottier** [comment]
Yep!

**Pull request merged**